GoogleBiddingAdapter/build.gradle.kts

```diff
@@ -34,7 +34,7 @@ android {
         minSdk = 21
         targetSdk = 33
         // If you touch the following line, don't forget to update scripts/get_rc_version.zsh
-        android.defaultConfig.versionName = System.getenv("VERSION_OVERRIDE") ?: "4.22.2.0.0"
+        android.defaultConfig.versionName = System.getenv("VERSION_OVERRIDE") ?: "4.22.1.0.1"
```

Comment:
you didn't change this part
implementation("com.google.android.gms:play-services-ads:22.2.0")